pyhocon/config_tree.py

```diff
@@ -205,8 +205,8 @@ def get_string(self, key, default=UndefinedKey):
             return None
 
         string_value = unicode(value)
-        if string_value in ['True', 'False']:
-            return string_value.lower()
+        if isinstance(value, bool):
```

Comment:
I had to keep the special casing for `bool` in `get_string`.

Consider the code from [this test case](https://github.com/chimpler/pyhocon/blob/8609e4f810ca47a3b573d8b79fa39760e97714b5/tests/test_config_tree.py#L162):

```python
config_tree = ConfigTree()
config_tree.put("bool-true", True)
assert config_tree.get_string("bool-true") == "true"
```

Here we insert a `bool` and expect a lower-case string back. Getting a lower-case string back is nice, since it is consistent with the string you would get [if you serialized the config_tree to HOCON and parsed it again](https://github.com/chimpler/pyhocon/blob/8609e4f810ca47a3b573d8b79fa39760e97714b5/pyhocon/tool.py#L68). So in the case of `get_string`, we need to have this handling.

Perhaps to keep it DRY, we could literally call [`to_hocon`](https://github.com/chimpler/pyhocon/blob/8609e4f810ca47a3b573d8b79fa39760e97714b5/pyhocon/tool.py#L68) on any `value` that is not already a string.

Thoughts?

Reply:
Thanks @movermeyer. I like that idea.